apps/ledger-live-desktop/src/renderer/families/evm/SendAmountFields/GasPriceField.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback } from "react";
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import { Account } from "@ledgerhq/types-live";
+import { Transaction, TransactionStatus } from "@ledgerhq/coin-evm/types";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import FeeSliderField from "~/renderer/components/FeeSliderField";
+import { Range, inferDynamicRange } from "@ledgerhq/live-common/range";
+type Props = {
+  account: Account;
+  transaction: Transaction;
+  status: TransactionStatus;
+  updateTransaction: (updater: (_: Transaction) => Transaction) => void;
+};
+const fallbackGasPrice = inferDynamicRange(BigNumber(10e9));
+let lastNetworkGasPrice: Range | undefined; // local cache of last value to prevent extra blinks
```

Comment:
If I remember correctly, this is directly coming from the old implementation of the Ethereum family, and I'm wondering if you tried testing if it was really necessary ? 
I think it's still very unclear what it actually does, the random `10e9` (1 gwei) is coming out of nowhere with no good reason for it, etc etc...

I would really love if we could challenge this and make sure this is useful for us, especially since the `gasOptions` is supposed to always exist if we're seeing this component.